frontend/src/scenes/billing/UnsubscribeSurveyModal.tsx

```diff
@@ -83,6 +85,7 @@ export const UnsubscribeSurveyModal = ({ product }: { product: BillingProductV2T
                             status={textAreaNotEmpty ? 'primary' : 'muted'}
                             onClick={() => {
                                 textAreaNotEmpty && reportSurveySent(surveyID, surveyResponse)
```

Comment:
Wait, wasn't it supposed to run `reportSurveyDismissed` here if there was no text? I think that was the point of this PR haha

```suggestion
                                textAreaNotEmpty ? reportSurveySent(surveyID, surveyResponse) : reportSurveyDismissed()
```

Reply:
You're right 🤦‍♀️ , thank you.